showcase/sankey/voronoi.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import Sankey from 'sankey';
+
+const BLURRED_NODE_OPACITY = 0.8;
+const FOCUSED_NODE_OPACITY = 1;
+
+const nodes = [{name: 'a'}, {name: 'b'}, {name: 'c'}];
+const links = [
+  {source: 0, target: 1, value: 10},
+  {source: 0, target: 2, value: 20},
+  {source: 1, target: 2, value: 20}
+];
+
+export default class VoronoiSankeyExample extends React.Component {
+  state = {
+    activeNode: null
+  }
+  render() {
+    const {activeNode} = this.state;
+
+    return (
+      <Sankey
+        nodes={nodes.map(d => ({
+          ...d,
+          opacity: activeNode && d.name === activeNode.name ? FOCUSED_NODE_OPACITY : BLURRED_NODE_OPACITY
+        }))}
+        links={links.map(d => ({...d}))}
+        width={200}
+        height={200}
+        hasVoronoi={true}
```

Comment:
https://github.com/uber/react-vis/blob/v0.10.3/src/sankey/index.js#L19 ?

It doesn't work when I remove the prop.